tests/test_issue_1514/test_rp_integration.py

```diff
@@ -0,0 +1,201 @@
+            'access_schema' : config[resource]['schema'],
+            'cores'         : config[resource]['cores'],
+            }
+    pdesc = rp.ComputePilotDescription(pd_init)
+
+    ## Launch the pilot.
+    pilot = pmgr.submit_pilots(pdesc)
+
+    ## Register the ComputePilot in a UnitManager object.
+    umgr = rp.UnitManager(session=session)
+    umgr.add_pilots(pilot)
+
+    cuds = list()      
+    
+    ## Test for single file 
+    cud = rp.ComputeUnitDescription()
+    cud.executable     = '/bin/date'
+    cud.input_staging  = {'source':os.path.join(local_sample_data, sample_data[0]), 
+                          'target':'unit:///%s' % sample_data[0],
+                          'action': rp.TRANSFER} 
+    cud.output_staging = {'source':'unit:///%s' % sample_data[0], 
+                          'target': './single_file_2.txt',
+                          'action': rp.TRANSFER}
+    
+  
+    cuds.append(cud)
+
+    ## Test for single folder
+    cud = rp.ComputeUnitDescription()
+    cud.executable     = '/bin/date'
+    cud.input_staging  = {'source':os.path.join(local_sample_data, sample_data[1]), 
+                          'target':'unit:///%s' % sample_data[1],
+                          'action': rp.TRANSFER} 
+
+    cud.output_staging = {'source':'unit:///%s' % sample_data[1], 
+                          'target': './single_folder_2',
+                          'action': rp.TRANSFER}
+    cuds.append(cud)
+
+    ## Test for multiple folder
+    cud = rp.ComputeUnitDescription()
+    cud.executable     = '/bin/date'
+    cud.input_staging  = {'source':os.path.join(local_sample_data, sample_data[2]), 
+                          'target':'unit:///%s' % sample_data[2],
+                          'action': rp.TRANSFER} 
+
+    cud.output_staging = {'source':'unit:///%s' % sample_data[2], 
+                          'target': './multi_folder_2',
+                          'action': rp.TRANSFER}                
+
+    cuds.append(cud)
+
+    ## Submit the previously created ComputeUnit descriptions to the
+    ## PilotManager. This will trigger the selected scheduler to start
+    ## assigning ComputeUnits to the ComputePilots.
+    units = umgr.submit_units(cuds)
+
+    ## Wait for all compute units to reach a final state (DONE, CANCELED or FAILED).
+    umgr.wait_units()
+    # ------------------------------------------------------------------------------------------------------------------
+  
+    # Extract info from RP config file
+    #-------------------------------------------------------------------------------------------------------------------
+    config_loc = '../../src/radical/pilot/configs/resource_%s.json'%resource.split('.')[0]
+    path_to_rp_config_file = os.path.realpath(os.path.join(os.getcwd(),config_loc))
+    cfg_file = ru.read_json(path_to_rp_config_file)[resource.split('.')[1]]
+    access_schema = config[resource]['schema']
+    if not access_schema:
+        access_schema = 'ssh'
+
```

Comment:
The above can be written as
```
access_schema = config[resource].get('schema', 'ssh')
```
Also, the resource configs are available on the session, so you don't need to parse them.  This test will break if RP changes config handling...

Reply:
Do you mean I can get the paths from the session? Can you elaborate on what you mean by 'RP changes config handling'? I don't understand how the test will break